planner/workers.go

```diff
@@ -0,0 +1,113 @@
+package planner
+
```

Comment:
so the workers are going to receive the Job and run it

Reply:
yes, they are running in the background and ready for tasks from the job queue.

Reply:
and each worker session is separate based on the user . So let's say we have `10` users logged in at the same time, and they all are triggering jobs we might have `100` jobs running on the backend which might be very rare right now, but trying to visualize ?